### PR TITLE
Blazor wasm example addition

### LINKED ISSUE #118: Blazor
By making a PR you are agreeing to that contributing a hello world project allows the repo/project owner to use the hello world app in examples for their own use.

- [ ] only the most simple basic hello world app
- [ ] remove all tests and commands for tests and dependencies for tests
- [ ] must say "Hello World" on the main component
- [ ] only include the minimal components (1 ideally)
- [ ] no routing, not extras whatsoever
- [ ] no styling
- [ ] nothing else !
- [ ] must work by running `npm install` and `npm start` (or equivalent)
- [ ] must be in the folder named `<tech-name>`
- [ ] add build steps and which folder to serve in the README.md
- [ ] verify that the app once built can be served from whatever folder the built assets are placed in
- [ ] verify that running `npx http-server <your-folder-to-serve>` works
- [ ] verify that a Pull Request doesnt already exist. If it does, feel free to contribute to it.

### PR DESCRIPTION
**Pull request by spboyer**
By making a PR you are agreeing to that contributing a hello world project allows the repo/project owner to use the hello world app in examples for their own use.

- [X] only the most simple basic hello world app
- [X] remove all tests and commands for tests and dependencies for tests
- [X] must say "Hello World" on the main component
- [X] only include the minimal components (1 ideally)
- [X] no routing, not extras whatsoever
- [X] no styling
- [X] nothing else !
- [X] must work by running `npm install` and `npm start` (or equivalent)
- [X] must be in the folder named `<tech-name>`
- [X] add build steps and which folder to serve in the README.md
- [X] verify that the app once built can be served from whatever folder the built assets are placed in
- [X] verify that running ~~`npx http-server <your-folder-to-serve>` works~~ `dotnet run` in this case
- [X] verify that a Pull Request doesn't already exist. If it does, feel free to contribue to it.

Closes #118

### CODE REVIEW

**Comment by johnpapa** [comment]
@spboyer please add the steps to the main readme in the root of the repo.

**Comment by manekinekko** [comment]
Thanks a lot @spboyer 

I understand the `dotnet` SDK is required to build the app. Do you think we can have it installed from npm? I found this package published by Microsoft, will it work: https://www.npmjs.com/package/dotnet-3.1?

**Comment by spboyer** [comment]
@manekinekko - this package is not an official Microsoft npm package and breakage would be likely.  I would not suggest installing it from npm no more that I'd recommend installing a node app from .NET

**Comment by manekinekko** [comment]
@spboyer gotcha!

@johnpapa would this be a blocker when deploying this Hello World app on SWA?

**Comment by spboyer** [comment]
@manekinekko Blazor support in SWA is not fully implemented. We are heads down getting that support completed.

File: blazor/Pages/Index.razor [hunk 1]
```diff
@@ -0,0 +1,2 @@
+@page "/"
+<div>Hello World!</div>
```
Comment (danroth27) [inline blazor/Pages/Index.razor]: Move this markup (without the page directive) into App.Razor and remove the Pages directory.

File: blazor/App.razor [hunk 2]
```diff
@@ -0,0 +1,9 @@
+@using Microsoft.AspNetCore.Components.Routing
+<Router AppAssembly="@typeof(Program).Assembly">
```
Comment (danroth27) [inline blazor/App.razor]: No routing allowed! :smile:

File: blazor/App.razor [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+    <Found Context="routeData">
+        <RouteView RouteData="@routeData" />
+    </Found>
+    <NotFound>
+        <p>Sorry, there's nothing at this address.</p>
+    </NotFound>
+</Router>
```
Comment (danroth27) [inline blazor/App.razor]:
```suggestion
<div>Hello, world!</div>
```

**Review by danroth27 — CHANGES_REQUESTED** [review]
Thanks for contributing this!

File: blazor/App.razor (hunk 3 above)
Comment (spboyer) [reply blazor/App.razor]: Thanks!

**Comment by spboyer** [comment]
@johnpapa @manekinekko - Ok to merge here.

**Review by manekinekko — APPROVED** [review]
Lgtm

**Comment by spboyer** [comment]
Still needed? @johnpapa

**Comment by manekinekko** [comment]
Thank you @spboyer. Merged.

**Pull request merged**